tb_plugin/torch_tb_profiler/profiler/module_parser.py

```diff
@@ -53,6 +53,7 @@ def __init__(self):
         self.op_list_groupby_name = []  # For Operator-view.
         self.op_list_groupby_name_input = []  # For Operator-view.
         self.kernel_list_groupby_name_op = {}  # For Kernel-view.
+        self.runtime_node_list = []  # For rule based recommendation of gpu metrics.
```

Comment:
this field is moved to nodeparser already

Reply:
Thanks. Fixed.